src/mpc-hc/MainFrm.cpp

```diff
@@ -1978,11 +1978,7 @@ void CMainFrame::OnTimer(UINT_PTR nIDEvent)
                             if (m_bRememberFilePos && !m_fEndOfStream) {
                                 auto* pMRU = &AfxGetAppSettings().MRU;

```

Comment:
Since persistedFilePosition is only relevant for current playing file, I would be more efficient to store it as a general value in the appsettings.